conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/DropRuleInterpreter.scala

```diff
@@ -15,21 +15,26 @@
 
 package za.co.absa.enceladus.conformance.interpreter.rules
 
-import za.co.absa.enceladus.model.conformanceRule.DropConformanceRule
-import org.apache.spark.sql.Dataset
-import org.apache.spark.sql.Row
-import org.apache.spark.sql.SparkSession
-import za.co.absa.enceladus.dao.EnceladusDAO
+import org.apache.spark.sql.{Dataset, Row, SparkSession}
 import za.co.absa.enceladus.conformance.CmdConfig
-import za.co.absa.enceladus.utils.transformations.{ArrayTransformations, DeepArrayTransformations}
+import za.co.absa.enceladus.dao.EnceladusDAO
+import za.co.absa.enceladus.model.conformanceRule.DropConformanceRule
+import za.co.absa.enceladus.utils.schema.SchemaUtils
+import za.co.absa.enceladus.utils.transformations.DeepArrayTransformations
 
 case class DropRuleInterpreter(rule: DropConformanceRule) extends RuleInterpreter {
 
   def conform(df: Dataset[Row])(implicit spark: SparkSession, dao: EnceladusDAO, progArgs: CmdConfig): Dataset[Row] = {
-    if (rule.outputColumn.contains('.')) {
-      conformNestedField(df)
-    } else {
-      conformRootField(df)
+    SchemaUtils.getFieldType(rule.outputColumn, df.schema) match {
```

Comment:
I thought there might already be a function to check if the path exists, but can't find it. It might be worth extracting an `exists` function.

Reply:
Will be done by 17:00 today.

Reply:
I am not sure if this is what you wanted. We basically keep everything the same. But it is true you don't have to convert to boolean in the end